features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/AutoSessionScenario.m

```diff
@@ -0,0 +1,20 @@
+//
+// Created by Jamie Lynch on 07/06/2018.
+// Copyright (c) 2018 Bugsnag. All rights reserved.
+//
+
+#import "AutoSessionScenario.h"
+
+
+@implementation AutoSessionScenario
+
+- (void)startBugsnag {
+    self.config.shouldAutoCaptureSessions = YES;
```

Comment:
Ordinarily this would be true, but the `config` object passed to the scenario has `shouldAutoCaptureSessions` set to false. This avoids needing to update all the other scenarios to anticipate an additional request.